replit/nixmodules
# C/C++ to use fileParam

Why
===

These feel out of place that they hardcode the entrypoint instead of using a parameter. The reason I did it this way are irrelevant now.

What changed
============

Use the file parameter so that the entrypoint specified in .replit will be used.

Test plan
=========

1. install the C or C++ module
2. make a main.c or main.cpp file
3. set it as the entrypoint in a repl
4. reload the repl
5. click run and see it run
